lib/web_apis/browser_api.es6.js

```diff
@@ -19,68 +19,82 @@
 foam.CLASS({
   name: 'BrowserAPI',
   package: 'org.chromium.apis.web',
-  documentation: `BrowserAPI is a FOAM model that contains browser name,
-  	browser version, OS, OS version, interface name and API name.`,
+  documentation: `BrowserAPI that contains browser name, browser version,
+  	 OS, OS version, interface name and API name.`,
+  ids: [
+    'browserName',
+    'browserVersion',
+    'osName',
+    'osVersion',
+    'interfaceName',
+    'apiName',
+  ],
   properties: [
     {
+      class: 'String',
       name: 'browserName',
+      documentation: `The name of browser.`,
       required: true,
       final: true,
-      class: 'String',
     },
     {
+      class: 'String',
       name: 'browserVersion',
+      documentation: `The version of browser.`,
+      documentation: ``,
       required: true,
       final: true,
-      class: 'String',
     },
     {
+      class: 'String',
       name: 'osName',
+      documentation: `The name of operating system.`,
       required: true,
       final: true,
-      class: 'String',
     },
     {
+      class: 'String',
       name: 'osVersion',
+      documentation: `The version of operating system.`,
       required: true,
       final: true,
-      class: 'String',
     },
     {
+      class: 'String',
       name: 'interfaceName',
+      documentation: `The name of an interface. Interfaces in browser
+        environment are identified as: first level or __proto__ object
+        that have meaningful properties.`,
       required: true,
       final: true,
-      class: 'String',
     },
     {
+      class: 'String',
       name: 'apiName',
+      documentation: `The name of an API. APIs are identified as meaningful
+        properties of an interface.`,
```

Comment:
As above: "meaningful property" should be defined in an explainer (and cited here).